platform/darwin/src/MGLTileID.h

```diff
@@ -0,0 +1,19 @@
+#import <Foundation/Foundation.h>
```

Comment:
If we intend to upstream this code, maybe add "/* */" style comments and description to make this code look like other .h/.m in https://github.com/trailbehind/mapbox-gl-native/tree/master/platform/darwin/src